src/Menu/Menu.js

```diff
@@ -103,6 +103,11 @@ class Menu extends React.Component {
     initiallyKeyboardFocused: React.PropTypes.bool,
 
     /**
+    * If true, the menu will be focused by the keyboard.
+    */
+    keyboardFocused: React.PropTypes.bool,
```

Comment:
Of course @oliviertassinari. :smile: 
One doubt, I put the default Value as `false`?
I'm afraid that could become a little confusing, since we are changing the value in the constructor, based on the on  props.initiallyKeyboardFocused.
What do you think?